app/src/main/res/layout/activity_contest_overview.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_contest_overview"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="io.intrepid.contest.screens.contestoverview.ContestOverviewActivity"
+    >
+
+    <Button
+        android:id="@+id/contest_overview_submit_button"
+        style="@style/AppTheme.BottomNavigationButton"
+        android:layout_alignParentBottom="true"
+        android:text="@string/contest_overview_submit_button_text"
+        />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:paddingBottom="@dimen/activity_vertical_margin"
```

Comment:
can we just make this a single android:padding attribute? I think activity_vertical_margin and horizontal have same dp value.